android/FacebookSDK400/build.gradle

```diff
@@ -8,7 +8,8 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.3'
-        
```

Comment:
can remove this unwanted change

Reply:
was added by android studio, will remove it

Reply:
done